specfile/specfile.py

```diff
@@ -407,15 +408,19 @@ def add_changelog_entry(
             email: E-mail of the author.
             timestamp: Timestamp of the entry.
               Supply `datetime` rather than `date` for extended format.
+            evr: Override the evr part of the changelog.
```

Comment:
Maybe it's better to capitalize the abbreviation:
```suggestion
            evr: Override the EVR part of the changelog.
```